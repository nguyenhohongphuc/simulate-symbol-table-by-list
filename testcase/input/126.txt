INSERT x number
ASSIGN x 123a