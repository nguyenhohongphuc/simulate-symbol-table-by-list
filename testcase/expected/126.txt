Invalid: ASSIGN x 123a